app/app.go

```diff
@@ -490,23 +490,8 @@ func NewSommelierApp(
 	// Setup an upgrade handler if doing an upgrade module upgrade
 	app.UpgradeKeeper.SetUpgradeHandler(
 		upgradeName,
-		func(ctx sdk.Context, _ upgradetypes.Plan, _ module.VersionMap) (module.VersionMap, error) {
-			// fromVM := make(map[string]uint64)
-			// for moduleName := range app.mm.Modules {
-			// 	fromVM[moduleName] = 1
-			// }
-			params := allocationtypes.DefaultGenesisState()
-			genstate, err := app.appCodec.MarshalJSON(&params)
-			if err != nil {
-				return nil, err
-			}
-			// json.RawMessage(genstate)
-			app.mm.Modules[allocationtypes.ModuleName].InitGenesis(ctx, app.appCodec, genstate)
-			// delete(fromVM, allocationtypes.ModuleName)
-
-			app.GravityKeeper.MigrateGravityContract(ctx, newGravityContractAddress, newGravityContractDeployHeight)
-
-			return nil, nil
+		func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
```

Comment:
If there is no upgrade in this release, should we be calling `SetUpgradeHandler` at all?